src/libraries/Common/src/Interop/Windows/BCrypt/Interop.BCryptSignHash.cs

```diff
@@ -111,5 +111,44 @@ internal static unsafe void BCryptSignHashPqcPure(
                 throw Interop.BCrypt.CreateCryptographicException(status);
             }
         }
+
+        internal static unsafe void BCryptSignHashPqcPreHash(
+            SafeBCryptKeyHandle key,
+            ReadOnlySpan<byte> hash,
+            string hashAlgorithmIdentifier,
+            ReadOnlySpan<byte> context,
+            Span<byte> destination)
+        {
+            NTSTATUS status;
+            int bytesWritten;
+
+            fixed (byte* pHash = &MemoryMarshal.GetReference(hash))
+            fixed (byte* pDest = &Helpers.GetNonNullPinnableReference(destination))
```

Comment:
Does this have to be `GetNonNullPinnableReference`? Empty destination should be rejected by caller.